ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPolicySimulatorSidePanel.tsx

```diff
@@ -373,17 +374,19 @@ function NetworkPolicySimulatorSidePanel({
     return (
         <Stack>
             <StackItem>
-                <Flex direction={{ default: 'row' }} className="pf-u-p-lg pf-u-mb-0">
-                    <FlexItem>
-                        <TextContent>
-                            <Text
-                                component={TextVariants.h2}
-                                className="pf-u-font-size-xl pf-u-mr-xl"
-                            >
-                                Simulate network policy for cluster “{scopeHierarchy.cluster.name}”
-                            </Text>
-                        </TextContent>
-                    </FlexItem>
+                <Flex
+                    direction={{ default: 'column' }}
+                    spaceItems={{ default: 'spaceItemsSm' }}
+                    className="pf-u-p-md pf-u-pb-sm pf-u-mb-0"
+                >
+                    <TextContent>
+                        <Text component={TextVariants.h2} className="pf-u-font-size-xl pf-u-mr-2xl">
+                            Generate network policies from the baselines of
```

Comment:
Is the suggestion just to keep "Generate network policies"? But that's not specific enough either.
@pedrottimark @dvail What about "Generate network policies from the baselines of the following entities:"

Reply:
Yes, that makes sense in the context.

A writer will have to make a judgment call how to refer to it in a step by step procedure, because even that will not make sense in a help page, except with a specific realistic example.

Is **of the following entities** a way to refer to what I saw somewhere else called the **scope**?

Reply:
yes, correct,  that the 'of the following entities' refer to the 'scope', it scope makes more sense then we can use the word scope instead of entities. It will then be 'of the following scope', or is this not what you mean by the suggestion?

Reply:
Thank you. I do not have a preference one versus the other, but here is suggestion from writer viewpoint.
* Pick one primary term for a concept.
* If it needs an explanation, instead of alternating between various synonyms, explicitly relate the term to its explanation in sentence text, as in the paragraph if you scroll down a short way. If there is an alternate widely used technical term, **also known as** is a helpful phrase in a sentence.

A possible benefit of **scope** is it allows alternatives (mentioning, not necessarily recommending)

Generate network policies
Scope of baseline:
* All deployments
    * 1 namespace
        * cluster: remote

Having written that, I wonder if **baseline** is a term whose meaning is obvious, or deserves concise explanation in the paragraph below.

Reply:
yes, this design aims to show the scope for the users so they know where the baselines come from. and thus, it makes more sense to emphasize/include the word 'baseline'.